pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -175,11 +175,11 @@ class ModuleDisplay extends Component<any, any, any> {
     private fetchModuleDetails = async(data) => {
         await this.getVariantParam()
         this.setState({
-            modulePath: data.location.pathname,
-            releasePath: "/content" + data.location.pathname + ".preview?variant=" + this.state.variant
+            modulePath: data.location.pathname.substring(7),
+            releasePath: "/content" + data.location.pathname.substring(7) + ".preview?variant=" + this.state.variant
         })
 
-        fetch(data.location.pathname + '/en_US.harray.4.json')
+        fetch(data.location.pathname.subtring(7) + '/en_US.harray.4.json')
```

Comment:
same as above comment

Reply:
I updated the code to make it more readable.